R/jobs.R

```diff
@@ -72,6 +72,9 @@ cloudml_train <- function(file = "train.R",
     cloudml$storage <- gcloud_project_bucket(project)
   }
 
+  # region is required
+  if (is.null(gcloud$region)) gcloud$region <- "us-east1"
```

Comment:
I think this will end up overriding the user's default region (as specified during `gcloud init`). I think we instead need to do this to read / check the default value:

```r
gcloud_default_regsion <- function() {
   trimws(gcloud_exec("config", "get-value", "region")$stdout)
}
```